src/redux/selectors/data/__specs__/stocksSelectors.spec.js

```diff
@@ -272,9 +273,41 @@ describe('selectBookables', () => {
         offerId: 'AA',
         userHasAlreadyBookedThisDate: true,
         userHasCancelledThisDate: false,
+        isBookable: true,
       },
     ])
   })
+
+  it('should not return stocks if isBookable is false', () => {
```

Comment:
```suggestion
  it('should not return stock when it is not bookable', () => {
```
Comme ça le nom parle du métier, et si la technique change demain il peut rester ainsi :)